package.json

```diff
@@ -59,9 +59,13 @@
     },
     "devDependencies": {
         "eslint": "^6.5.1",
-        "prettier": "1.18.2"
+        "husky": "3.0.9",
+        "prettier": "1.18.2",
+        "pretty-quick": "2.0.0"
     },
-    "prettier": {
```

Comment:
Can we still have a `prettier` key in `package.json` or an empty `.prettierrc` file in the repo? I use the presence of [prettier config](https://prettier.io/docs/en/configuration.html) to turn on automatic formatting in my editor.

Reply:
Done.